apps/omg_db/lib/omg_db/rocksdb/server.ex

```diff
@@ -61,18 +63,40 @@ defmodule OMG.DB.RocksDB.Server do
       |> String.to_atom()
 
     {:ok, _recorder_pid} = Recorder.start_link(%Recorder{name: recorder_name, parent: self(), table: table})
-    setup = [{:create_if_missing, true}, {:prefix_extractor, {:fixed_prefix_transform, 1}}]
+    setup = [{:create_if_missing, false}, {:prefix_extractor, {:fixed_prefix_transform, 5}}]
```

Comment:
hm, `{:create_if_missing, false}` is a tiny bit surprising, why?

Reply:
hmm.. this is the `init`, so you had to call `init_storage` first already, no? thats how ldb works.

Reply:
ah, I forgot about that split. I see `leveldb/server` has it like this, thx & nvm